MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -31,7 +31,7 @@ import Foundation
 
     let authTokens: MSALNativeAuthTokens
     let configuration: MSALNativeAuthConfiguration
-    private let cacheAccessor: MSALNativeAuthCacheInterface
+    let cacheAccessor: MSALNativeAuthCacheInterface
```

Comment:
why this property isn't private anymore?

Reply:
It was because it was used in an "internal" method in an extension. I have updated the method to make it a parameter and put it back to being private.